### ISSUE TITLE
Switch to 64 bit System.

### ISSUE THREAD

**Issue by chtheitmmmmm** [hm-riscv/vscode-riscv-venus#106]
When I write 
ld x21, 8(x22)  
It throws "LD instruction is not supported on 32bit System".
Does there exist any way to switch to 64 bit?

**Comment by daniel-pg** [comment]
This extension uses [Venus RISC-V Simulator](https://github.com/ThaumicMekanism/venus), which currently doesn't support simulating the RV64 architecture and its instructions. In fact, it doesn't even implement the full RV32 specification, only the Integer(I) base instruction set and the Multiplication(M) extension.

**Comment by Wladefant** [comment]
You are correct although it is based on this fork https://github.com/61c-teach/venus
